mpc-core/src/protocols/plain.rs

```diff
@@ -193,12 +241,172 @@ impl<F: PrimeField> PrimeFieldMpcProtocol<F> for PlainDriver<F> {
         dst[dst_offset..dst_offset + len].clone_from_slice(&src[src_offset..src_offset + len]);
     }
 
+    fn print_share(&self, to_print: &Self::FieldShare) {
+        if to_print.is_zero() {
```

Comment:
why do we need this `is_zero`?

Reply:
is required since ark_ff does not print anything for a 0, and that is annoying.